src/panel/pages/events/components/TimelineDuration.tsx

```diff
@@ -9,10 +9,13 @@ export const TimelineAliveDuration = styled.div`
 
 type NetworkState = "fetching" | "success" | "error";
 
-export const NetworkDuration = styled.div`
+export const NetworkDuration = styled.div<{ isSelected?: boolean }>`
   cursor: pointer;
   height: 10px;
 
+  outline: ${({ isSelected, theme }) =>
```

Comment:
I think this looks good. The `TimelineSourceIcon` (`Q`/`M`) and the `TimelineEvent` use an onHover colour change instead of an outline. Part of me thinks we should be consistent with one or the other?

Reply:
My understanding is that we want to indicate the selected event whilst the user then inspects the event in other parts of the UI. 
So the outline needs to persist beyond the hover (I see that the demo gif doesn't make that clear).

Reply:
Oh yeah you're right it has to persist after hover - my thinking was that maybe we could use a colour change instead of an outline to align with them?

I don't have particularly strong feelings though, I think it looks good as it is

Reply:
Oh see! Sorry I misunderstood. I went with an outline so that the colour for `fetching | success| error` stays the same. My thinking is that the outline indicates selection and not the `data-state` of the event?

Reply:
Yea that makes sense - lgtm!